samples/Common.c

```diff
@@ -2,6 +2,9 @@
 #include "Samples.h"
 
 PSampleConfiguration gSampleConfiguration = NULL;
+CHAR pPeerConnectionMetricsMessage[MAX_PEER_CONNECTION_METRICS_MESSAGE_SIZE];
```

Comment:
Cant we have these in `pSampleStreamingSession` since you already are using it in the callback? 
As a rule of thumb, we should avoid using globals unless it is absolutely necessary.

Reply:
Sounds good! Moved inside streamingSession